packages/corvid-cli/test/setupJest.js

```diff
@@ -1 +1 @@
-jest.setTimeout(25000);
+jest.setTimeout(40000);
```

Comment:
anything specific required this change ?

Reply:
cli e2e tests on windows caused timeouts, after that I extended the timeout it passed